src/styles/components/_header.scss

```
@@ -0,0 +1,26 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+
+  &__left {
+    display: flex;
+    align-items: center;
```

Could we maintain alphabetic order in css properties?
```suggestion
    align-items: center;
    display: flex;
    
```